scene/gui/item_list.cpp

```diff
@@ -314,7 +314,7 @@ void ItemList::move_item(int p_item,int p_to_pos) {
 
 	if (current<0) {
 		//do none
-	} if (p_item==current) {
+	} else if (p_item==current) {
```

Comment:
That actually changes the logic, but I guess the previous behaviour could be considered a bug.